options/posix/include/unistd.h

```diff
@@ -268,6 +268,7 @@ size_t confstr(int __name, char *__buf, size_t __size);
 char *ctermid(char *__s);
 int dup(int __fd);
 int dup2(int __src_fd, int __dest_fd);
+int dup3(int __fd, int __newfd, int __flags);
```

Comment:
This needs to be guarded with appropriate macros, most likely `_POSIX_C_SOURCE` and `_GNU_SOURCE`.